src/ontology/mondo-edit.obo

```diff
@@ -574661,6 +574661,18 @@ is_a: MONDO:0022800 {source="https://clinicalgenome.org/affiliation/40065/", sou
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7379" xsd:anyURI
 
+[Term]
+id: MONDO:1040030
+name: GBA1-related Parkinson disease, susceptibility
+def: "Increased susceptibility for developing Parkinson disease due to the presence of a monoallelic pathogenic variant, likely pathogenic variant, or risk allele in the GBA1 gene." [https://orcid.org/0000-0003-3254-7564, https://search.clinicalgenome.org/CCID\:004931, PMID:19433656, PMID:19846850, PMID:19858467]
```

Comment:
This term should follow this pattern:
https://github.com/monarch-initiative/mondo/blob/master/src/patterns/dosdp-patterns/susceptibility_by_gene.yaml

Could you please update the definition to follow this pattern (and feel free to modify it to include the information provided by ClinGen)

Reply:
I think I did it right?

Reply:
looks good, thanks!